src/Makefile.am

```diff
@@ -56,9 +56,6 @@ if SQLITE
 sqlite_sources = ReadStreamSQLite.h WriteStreamSQLite.h
 endif
 
-if MPI
```

Comment:
Also remove the mpi test

Reply:
Also please reformat the source code in the src directory:
```
clang-format-7 -i *.cpp *.h
```